rails-commenting.rb

```diff
@@ -7,24 +7,29 @@
 # FILE: app/controller/blog_posts_controller.rb
 
 # ---1)
+# creates an association with blog_posts_controller and application
 class BlogPostsController < ApplicationController
   def index
     # ---2)
+    # creates class index which is the main spot for all blog posts, sort of like a homepage
```

Comment:
I think you are referring to the line above which is a method index. The line below per instructions is the instance variable post's which will be an array of objects that hold all the instances from your database.

Reply:
"per the instructions" sounds weird....